teps/0044-decouple-task-composition-from-scheduling.md

```diff
@@ -504,156 +705,74 @@ Cons:
 * Might be hard to reason about what is executed together
 * Might be hard to reason about what which Tasks can be combined in a group and which can't
 
-#### some other directive, e.g. labels, to indicate what should be scheduled together?
+####### some other directive, e.g. labels, to indicate what should be scheduled together?
 
 This option is the same as the previous `groups` proposal but maybe we decide on some other ways to indicating grouping,
 e.g. labels.
 
-### Runtime instead of authoring time
+###### Focus on workspaces
```

Comment:
This make the assumption we want to solve a "workspace" problem, while, I think the TEP is about "Decouple Task Composition from Scheduling" (even if you remove any workspace in your tasks/pipeline, you might want to decouple composition from scheduling)